posix.mak

```diff
@@ -525,6 +525,17 @@ style: ../dscanner/dsc $(LIB)
 	@echo "Enforce space between binary operators"
 	grep -nrE "[[:alnum:]](==|!=|<=|<<|>>|>>>|^^)[[:alnum:]]|[[:alnum:]] (==|!=|<=|<<|>>|>>>|^^)[[:alnum:]]|[[:alnum:]](==|!=|<=|<<|>>|>>>|^^) [[:alnum:]]" $$(find . -name '*.d'); test $$? -eq 1
 
+	@echo "Validate changelog files (Do _not_ use REF in the title!)"
+	@echo " - Title line can't contain REF - it's already a link"
+	@echo " - After the title line an empty, separating line is expected"
+	@echo " - Which is supposed to be followed by a long description"
```

Comment:
Not a fan of all this noise, could you please only print if if the validation fails?